docs/resources/samples/conditional-formatting-samples.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Conditional formatting samples
+description: A collection of Office Scripts that use different Excel conditional formatting options.
+ms.date: 12/28/2023
+ms.localizationpriority: medium
+---
+
+# Conditional formatting samples
+
+Conditional formatting in Excel applies formatting to cells based on specific conditions or rules. This feature helps you visually highlight important data, identify trends, and analyze patterns in your spreadsheet. With conditional formatting, you'll quickly use color scales, data bars, and icon sets, to dynamically format your data. This page contains a collection of Office Scripts that demonstrate various conditional formatting options.
```

Comment:
The wording of this paragraph sounds like it is addressed to end users of Excel and tells them what conditional formatting can do for them. Should it be reworded to address the makers who are making an Excel Script?